utils/release_helper.py

```diff
@@ -0,0 +1,791 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""
+Script created to help maintainers during the release process by automating Github tasks.
+"""
+
+# References:
+# - https://developer.github.com/v3/libraries/
+# - https://github.com/PyGithub/PyGithub
+# - https://pygithub.readthedocs.io/en/latest/github_objects.html
+# - https://docs.github.com/en/rest
+
+from datetime import datetime, timedelta
+from github import Github
+import argparse
+import configparser
+import os.path
+import re
+import subprocess
+
+
+def get_parameter_from_ini(config_file, section, parameter) -> str:
+    config = configparser.ConfigParser()
+    try:
+        config.read(config_file)
+        return config[section][parameter]
+    except Exception as e:
+        print(f'Error: {e} entry was not found!')
+
+
+def create_ini_file_template(creds_file):
+    try:
+        new_creds_file = os.path.expanduser(creds_file)
+        with open(new_creds_file, mode='w', encoding='utf-8') as file:
+            file.write(
+                '[DEFAULT]\n'
+                'github_token = <generate your token in https://github.com/settings/tokens>\n')
+        print(f'Great! {new_creds_file} was created! Edit it to include your personal token.')
+    except Exception as e:
+        print(f'Error: {e}')
+
+
+def get_github_token(creds_file) -> str:
+    creds_file = os.path.expanduser(creds_file)
+    if os.path.exists(creds_file):
+        return get_parameter_from_ini(creds_file, "DEFAULT", "github_token")
+    else:
+        print(f'{creds_file} file was not found!')
+        if get_confirmation(f'Would you like to create the "{creds_file}" file?'):
+            create_ini_file_template(creds_file)
+
+
+def create_github_session(creds_file) -> object:
+    token = get_github_token(creds_file)
+    if not token:
+        print('WARNING: No token found. The API queries are very limited without a token.\n')
+    return Github(token)
+
+
+def get_confirmation(question) -> bool:
+    answer = str(input(f'{question} (Y/N): ')).lower().strip()
+return answer[:1] == 'y'
+        return True
```

Comment:
I think following three lines should not stay here. One return statement is enough.

Reply:
And indentation is wrong.

Reply:
I committed the review suggestion directly on Github interface. I will just amend the last commit fixing this.